single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultLabelProvider.java

```diff
@@ -130,41 +130,25 @@ public List<NodeDecorator> getNodeDecorators(Node node, Direction direction) {
 
         List<NodeDecorator> nodeDecorators = new ArrayList<>();
 
-        // OperatingStatus extension is on identifiables, so we're looking for them
-        if (node instanceof EquipmentNode && !(node instanceof SwitchNode)) {
-            if (node instanceof FeederNode) {
-                FeederNode feederNode = (FeederNode) node;
+        if (node instanceof EquipmentNode equipmentNode && !(node instanceof SwitchNode)) {
+            if (node instanceof FeederNode feederNode) {
                 switch (feederNode.getFeeder().getFeederType()) {
-                    case BRANCH:
-                    case TWO_WINDINGS_TRANSFORMER_LEG:
-                        Connectable<?> connectable = network.getConnectable(feederNode.getEquipmentId());
-                        if (connectable != null) {
-                            addBranchStatusDecorator(nodeDecorators, node, direction, connectable);
-                        }
-                        break;
-                    case THREE_WINDINGS_TRANSFORMER_LEG:
+                    case BRANCH, TWO_WINDINGS_TRANSFORMER_LEG -> addOperatingStatusDecorator(nodeDecorators, node, direction, network.getBranch(feederNode.getEquipmentId()));
+                    case THREE_WINDINGS_TRANSFORMER_LEG -> {
                         // if this is an outer leg (leg corresponding to another voltage level), we display the decorator on the inner 3wt
                         if (node.getAdjacentNodes().stream().noneMatch(Middle3WTNode.class::isInstance)) {
-                            addBranchStatusDecorator(nodeDecorators, node, direction, network.getThreeWindingsTransformer(feederNode.getEquipmentId()));
+                            addOperatingStatusDecorator(nodeDecorators, node, direction, network.getThreeWindingsTransformer(feederNode.getEquipmentId()));
                         }
-                        break;
-                    case HVDC:
-                        HvdcLine hvdcLine = network.getHvdcLine(feederNode.getEquipmentId());
-                        Connectable<?> converterStation = ((FeederWithSides) feederNode.getFeeder()).getSide() == NodeSide.ONE ? hvdcLine.getConverterStation1() : hvdcLine.getConverterStation2();
-                        addBranchStatusDecorator(nodeDecorators, node, direction, converterStation);
-                        break;
-                    default:
-                        break;
+                    }
+                    case HVDC -> addOperatingStatusDecorator(nodeDecorators, node, direction, network.getHvdcLine(feederNode.getEquipmentId()));
+                    default -> { /* Do nothing */ }
```

Comment:
It was a code smell before because you did nothing without telling why. Now that you added the comment it ensures that it was not an empty statement by mistake. But it's more valuable to also add more context like "no decorator for other feeder types"